.github/workflows/samples.yaml

```diff
@@ -12,3 +16,65 @@ jobs:
       - name: Run checkstyle
         run: mvn -P lint --quiet --batch-mode checkstyle:check
         working-directory: samples/snippets
+
+  test:
+    if: ${{ github.event.action != 'labeled' || github.event.label.name == 'actions:force-run' }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'write'
+      pull-requests: 'write'
+      id-token: 'write'
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{github.event.pull_request.head.ref}}
+        repository: ${{github.event.pull_request.head.repo.full_name}}
+    - uses: 'google-github-actions/auth@v0.4.2'
+      with:
+        workload_identity_provider: 'projects/1016721519174/locations/global/workloadIdentityPools/github-actions-pool/providers/github-actions-provider'
+        service_account: 'java-docs-samples-testing@java-docs-samples-testing.iam.gserviceaccount.com'
+        create_credentials_file: 'true'
+        access_token_lifetime: 300s
+        token_format: 'access_token'
+        cleanup_credentials: false
+    - id: 'secrets'
+      uses: 'google-github-actions/get-secretmanager-secrets@v0'
+      with:
+        secrets: |-
+          env_vars:cloud-devrel-kokoro-resources/java-bigquerystorage-samples-secrets
```

Comment:
With this strategy, we can also put this secret in the java-docs-samples-testing project instead of the shared project